src/js/app.js

```diff
@@ -4,16 +4,25 @@ import { App, Logo, GithubCorner, ChangeButton, Rebus } from './components';
 import { actions } from './store';
 import '../css/main.css';
 
+export const onKeyUp = event => {
+  switch (event.key) {
+    case 'Left':
+    case 'ArrowLeft':
+      return actions.prev();
+
+    case ' ':
+    case 'Spacebar':
+    case 'Right':
+    case 'ArrowRight':
+      return actions.next();
+
+    default:
+      return undefined;
+  }
+};
+
 export function registerListeners() {
-  document.addEventListener('keyup', event => {
-    const key = event.key || event.keyCode; // For older browser support
-    if (key === 'ArrowRight' || key === 39) {
```

Comment:
I think we still want to fall back to the keyCode for support in older browsers etc. You can just add `key === ' ' || key === 32` in the first if statement

Reply:
👍